payments/swish/other-features.md

```diff
@@ -493,6 +493,29 @@ Content-Type: application/json
 }
 ```
 
+### `swishgatewaytimeout`
+
+Caused By:
+
+- During a create a sale call to e-com, Swish responded with 504 (Gateway Timeout).
+
+{:.code-header}
+Example response swishgatewaytimeout
+
+```http
+HTTP/1.1 504 Gateway Timeout
+Content-Type: application/json
+
+{
+    "sessionId": "570ad610-3bd5-43d2-a270-ca1510562972",
+    "type": "https://api.payex.com/psp/errordetail/swishgatewaytimeout",
```

Comment:
Isn't it common to use the payment instrument as its own path segment to namespace instrument-specific problem types like so?

```suggestion
    "type": "https://api.payex.com/psp/errordetail/swish/gatewaytimeout",
```

Reply:
Looks like this might be a copy-paste error on this page. The error type in this case is actually `swishgatewaytimeout` but the prefix path might be incorrect yes.

Reply:
Looked it up in Octopus and basepath currently seems to be correct with what we have in production at the moment

Reply:
Ok. Since `swish` is used as an unseparated prefix in other problem `type` URIs already, I guess we will just have to accept that this inconsistency (or bug) will have to be dealt with later.